index.html

```diff
@@ -2612,12 +2647,15 @@ <h5>Existing implementations</h5>
 <dt>mapbox-embed</dt>
 <dt>leaflet-api</dt>
 <dt>open-layers-api</dt>
-<dt>tomtom-sdk</dt>
 <dt>bing-maps-api</dt>
 <dt>apple-mapkit-js-api</dt>
 <dd>
   <a>full support</a>
 </dd>
+<dt>tomtom-sdk</dt>
+<dd>
+  <a>supported, with limitations</a>: supports WMS services, but not TMS
```

Comment:
Since this capability is specifically about custom *tiles*, I'd call this no support, although it's still worth including the extra notes because you're likely not the only person to see "WMS" in the docs & assume that means you can use custom map servers.

WMS and TMS are defined terms earlier in the document, so I've added the href-less `<a>` elements to auto link. But I've also added some more in-place explanation of what the terms mean.

```suggestion
  <a>no support</a>: Although TomTom supports <a>WMS</a> services (where the server can generate a single image file for custom map coordinates),
      it does not support <a>TMS</a> or other custom tile sources
```